proposals/0311-task-locals.md

```diff
@@ -145,13 +145,13 @@ await Task.withLocal(\.requestID, boundTo: "1234-5678") {
 func nested() async {
   await nestedAgain() // "1234-5678"
   
-  await Task.withLocal(\.requestID, boundTo: "xxxx-zzzz") { 
+  Task.withLocal(\.requestID, boundTo: "xxxx-zzzz") { 
```

Comment:
Binding operations must be async -- notice that inside the closure we call an async function, so the withLocal must be async

```suggestion
  await Task.withLocal(\.requestID, boundTo: "xxxx-zzzz") { 
```